tools/ci.py

```
@@ -245,9 +247,40 @@ def install_slangpy_torch(args: Any):
+        out = run_command(cmd, env=env)
+    except RuntimeError as e:
+        out = getattr(e, "captured_output", "")
+        raise
+    finally:
+        if in_gha:
+            print("::endgroup::")
+            sys.stdout.flush()
+        # Surface pip's own pass/fail summary outside the collapsed group
+        # so the result is visible without expanding.
+        for line in out.splitlines():
+            if line.startswith("Successfully installed") or line.startswith("ERROR"):
```

Ruff (0.15.14)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)